lib/snippet/snippet.module.css

```diff
@@ -2,4 +2,32 @@
   font-family: monospace;
   font-size: 20px;
   margin-bottom: 20px;
-}
\ No newline at end of file
+  position: relative;
+}
+
+.copyButton:focus {
+  background-color: #fd0;
+  color: black;
+}
+
+.copyButton,
+.copyButton:hover {
```

Comment:
Doesn't this mean select all `.copyButton` _and_ `.copyButton:hover` elements? If so, I don't think we need the second selector.

Also, can we used kebab-case please? :)

Reply:
yes, that is the style for copyButton and copyButton:hover both. We need the second as well otherwise when hovering on the button the default gds-button:hover styling is used

Reply:
Done